src/HotChocolate/ApolloFederation/src/ApolloFederation/Properties/FederationResources.resx

```diff
@@ -159,4 +159,10 @@
   <data name="FieldDescriptorExtensions_Provides_FieldSet_CannotBeNullOrEmpty" xml:space="preserve">
     <value>Value cannot be null or empty.</value>
   </data>
+  <data name="ThrowHelper_EntityType_NoEntities" xml:space="preserve">
+    <value>The schema has no types with a KeyDirective and therefor no entities. For Apollo federation at least on entity is required.</value>
```

Comment:
I think active would be better, but i could be wrong:
... Apollo federation requires at least on entity.

Reply:
Adapted message accordingly.